cpp/lib/src/ZoteroHarvesterConfig.cc

```diff
@@ -339,7 +338,6 @@ JournalParams::JournalParams(const IniFile::Section &journal_section, const Glob
     ssgn_ = journal_section.getString(GetIniKeyString(SSGN), "");
     license_ = journal_section.getString(GetIniKeyString(LICENSE), "");
     selective_evaluation_ = journal_section.getBool(GetIniKeyString(SELECTIVE_EVALUATION), false);
-    force_language_detection_ = journal_section.getBool(GetIniKeyString(FORCE_LANGUAGE_DETECTION), false);
```

Comment:
This would stop the evaluation of `force_language_detection` in the config file. I thought these were two different aspects: Use `force_language_language_detection`to overrule an erroneous language code from the Translator and use the detected Tika result, whereas `*` is used to force a language known and set beforehand. If they are equal reintroducing `*` would add no further functionality as far as I see.

Reply:
The corresponding function is still called when evaluating the config file, see ParseExpectedLanguages on line 347. The `*` means that the language detection result will have priority over the language delivered by the Zotero Translator and therefore no error will be generated, see ZoteroHarvesterConversion.cc line 582 and following.

Reply:
But then there is no semantic difference to using `force_language_detection`?

Reply:
Thats correct - force_language_detection has just been moved to the LanguageSettings struct, because all parameters that need to be parsed by the ParseExpectedLanguages function belong there. The functionality is the same as with the old flag, except the old flag has been entirely disabled until now.